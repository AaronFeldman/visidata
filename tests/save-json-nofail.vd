sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.jsonl	o	
test			columns-sheet		C	
test_columns	name		sort-asc		[	
test_columns			quit-sheet		q	
test	amt		key-col		!	
test	key1		key-col		!	
test			sort-keys-asc		g[	
test	amt		key-col		!	
test	key1		key-col		!	
test			save-sheet	tests/golden/save-json-nofail.csv	^S	
test			save-sheet	tests/golden/save-json-nofail.json	^S	
test			save-sheet	tests/golden/save-json-nofail.md	^S	
test			save-sheet	tests/golden/save-json-nofail.html	^S	
test			save-sheet	tests/golden/save-json-nofail.txt	^S	
