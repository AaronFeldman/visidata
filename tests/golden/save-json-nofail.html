<h2 class="sheetname">%s</h2>
<table id="test">
<tr><th>amt</th><th>key1</th><th>key2</th><th>qty</th></tr>
<tr><td>{&quot;key1&quot;: &quot;2017-12-25 8:43&quot;, &quot;foo&quot;, &quot;amt&quot;: &quot;.03&quot;, &quot;qty&quot;: null}
</td><td>{&quot;key1&quot;: &quot;2017-12-25 8:43&quot;, &quot;foo&quot;, &quot;amt&quot;: &quot;.03&quot;, &quot;qty&quot;: null}
</td><td>{&quot;key1&quot;: &quot;2017-12-25 8:43&quot;, &quot;foo&quot;, &quot;amt&quot;: &quot;.03&quot;, &quot;qty&quot;: null}
</td><td>{&quot;key1&quot;: &quot;2017-12-25 8:43&quot;, &quot;foo&quot;, &quot;amt&quot;: &quot;.03&quot;, &quot;qty&quot;: null}
</td></tr>
<tr><td></td><td>2016-01-01 11:00:00</td><td>foo</td><td>1</td></tr>
<tr><td></td><td>2018-07-27 16:44</td><td>bar</td><td></td></tr>
<tr><td>.01</td><td>2018-07-27 18:44</td><td>baz</td><td>256</td></tr>
<tr><td>.01</td><td>2018-10-20 18:44</td><td>foo</td><td>30.0</td></tr>
<tr><td>.3</td><td>2017-12-25 8:44</td><td>bar</td><td>16</td></tr>
<tr><td>3</td><td>2016-01-01 1:00</td><td></td><td>2</td></tr>
<tr><td>3.3</td><td></td><td>baz</td><td>32</td></tr>
<tr><td>43.2</td><td></td><td>baz</td><td>4</td></tr>
<tr><td>9.1</td><td>2018-07-27 4:44</td><td></td><td>64</td></tr>
</table>